content/en/blog/_posts/XXXX-XX-XX-dra-133-update.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+title: "Kubernetes 1.33: New features in DRA"
```

Comment:
nit:
```suggestion
title: "Kubernetes v1.33: New features in DRA"
```

Reply:
Thanks! I've updated the PR to use the `v` prefix for all mentions of Kubernetes versions and also use lowercase for all uses of `alpha` and `beta`. I also removed the capitalization of alpha and beta in the headers as the document doesn't otherwise use title case (and looking through previous blog post there doesn't seem to be a clear guidance on whether we should).

Reply:
Thank you! I couldn't find a written convention for the feature stages either, but I see them mentioned in lowercase most times.

Reply:
This was something we touched on in one of the SIG Docs or SIG Release meetings (I can't remember where it was), but I don't think it's clearly documented... This should be a part of some guideline indeed!